### PR TITLE
Cleanup+improve complete distribution

### PR DESCRIPTION
**Pull request by okurz**
* Remove obsolete non-install-only modules
* Delete obsolete 'update' tests
* main: Supply extension automatically to loadtest-calls
* Update and consolidate code style

Related progress issue: https://progress.opensuse.org/issues/133025

### CODE REVIEW

File: tests/containers/single_container_webui.pm [hunk 1]
```diff
@@ -1,24 +1,23 @@
-use Mojo::Base qw(openQAcoretest);
+use qw(openQAcoretest);
```
Comment (perlpunk) [inline tests/containers/single_container_webui.pm:1]: What was the intent here?

Comment (okurz) [reply tests/containers/single_container_webui.pm]: oops, that should still have the Mojo::Base, or at least "base". Fixed

File: tests/osautoinst/worker.pm [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
     wait_for_desktop;
-    send_key 'ctrl-alt-f3';
+    switch_to_root_console;
     assert_screen 'inst-console';
     type_string "root\n";
-    assert_screen 'password-prompt', 10;
-    type_string $testapi::password . "\n";
+    assert_screen 'password-prompt';
+    type_password;
+    send_key 'ret';
```
Comment (kalikiana) [inline tests/osautoinst/worker.pm:13]: I guess what we really want here is this non-existing command:

```suggestion
    enter_password;
```

In that sense I find both the old and your new version equally desirable

**Review by kalikiana — APPROVED** [review]

**Review by perlpunk — APPROVED** [review]

File: scenario-definitions.yaml [hunk 3]
```diff
@@ -19,34 +19,25 @@ machines:
...
       LOAD_PYTHON_TEST_MODULES: "0"
   openqa_from_bootstrap:
     <<: *common
     settings:
-      <<: *common_settings
       OPENQA_FROM_BOOTSTRAP: "1"
   openqa_install:
     <<: *common
     settings:
-      <<: *common_settings
```
Comment (perlpunk) [inline scenario-definitions.yaml:47]: `settings` is empty here now, so it gets loaded as `null`. I will make a PR to fix it.

**Pull request merged**